acqa-repo1-aws-tf12-part1.tf

```diff
@@ -416,12 +438,12 @@ resource "aws_lb" "acqa-test-alb1" {
   security_groups    = [aws_security_group.acqa-test-securitygroup1.id]
   subnets            = [aws_subnet.acqa-test-albsubnet1.id, aws_subnet.acqa-test-albsubnet2.id]
 
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure IMDv1 is disabled for AWS EC2 instances in AWS Launch Configuration**
 
**Vulnerability**
AWS Launch Configuration that have IMDv1 enabled makes the end instances vulnerable to server side request forgery (SSRF) attacks.
 
**Remediation**
IMDv1 is vulnerable to SSRF attack. Thus, EC2 Instance attached to an AWS Launch Configuration should have metadata_options block with either http_endpoint attribute set to disabled or ensure http_tokens attribute is set to required. This ensures IMDv2 is being used.

source: accurics-